src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -60,9 +60,8 @@
 
     public Subsystems subsystems;
 
-    public Controls(Subsystems s) {
-
-        subsystems = s;
+    private Controls() {
+        subsystems = Subsystems.instance;
```

Comment:
Singletons are a classic anti-pattern, so I'm not going to approve more of their use.